Sources/Sentry/SentryANRTracker.m

```diff
@@ -9,6 +9,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static BOOL _appIsHanging = NO;
```

Comment:
This information was retrieved from a static property, allowing us to perform checks from any location without requiring a reference from the currently running tracker instance, technically the app is either hanging or not, does not matter how many trackers instances we have. However this is now irrelevant since this variable has been removed.

Reply:
The variable is still in the PR, @brustolin. Did you maybe forget to push 🤔 ?

Reply:
I was pushing and notice something else to fix. Sorry about this.